src/Config.cpp

```diff
@@ -71,7 +70,7 @@ void          _activateConfig();
 // ************************************************************
 // configBuffer handling
 // ************************************************************
-// reads the EEPROM until NUL terminator and returns the number of characters incl. terminator, starting from given address
+// reads the EEPRROM until NULL termination and returns the number of characters incl. NULL termination, starting from given address
```

Comment:
Not really worth mentioning, but "EEPRROM" is spelled wrong, and the "\0" char is a NUL char used as terminator; NULL is something that has "no" value (or the "placeholder" value representing that).